pkgs/tools/misc/gams/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, unzip, lib, file, license, optgams }:
+
+stdenv.mkDerivation rec {
+  version = "25.0.2";
+  name = "gams-${version}";
+  src = fetchurl {
+    url = "https://d37drm4t2jghv5.cloudfront.net/distributions/${version}/linux/linux_x64_64_sfx.exe";
+    sha256 = "4f95389579f33ff7c2586838a2c19021aa0746279555cbb51aa6e0efd09bd297";
+  };
+  unpackCmd = "unzip $src";
+  buildInputs = [ unzip file ];
+  dontBuild = true;
+  binaries = [ "decisc" "gmsbncux.out" "eigenvalue" "mcfilter" "gmsptnux.out" "csv2gdx" "gdx2veda" "endecrypt" "eigenvector" "gmsderux.out" "gmsba_ux.out" "gmsdi_ux.out" "gmssb_ux.out" "sqlite3" "gamskeep" "gmsck_ux.out" "gmsgewux.out" "finlib" "apilib" "noalib" "gmsls_ux.out" "gmsdetux.out" "gmsxa_ux.out" "gmscr_ux.out" "gmsunzip" "scenred2" "mps2gms" "gdxrename" "gmscvpux.out" "gmsckbux.out" "bib2gms" "gmsbaxux.out" "gmszip" "gmscvlux.out" "decism" "gmsecpux.out" "gmsbd_ux.out" "gdxdiff" "gamscmex.out" "gamsinst" "invert" "gmscv_ux.out" "chk4upd" "gdxmerge" "gmscvnux.out" "gmsmd_ux.out" "cholesky" "gmsunpack" "gdxdump" "gmsge_ux.out" "gamslib" "gdxcopy" "gmsgenux.out" "gmsex_ux.out" "gdx2sqlite" "hexdump" "gmsja_ux.out" "datalib" "gmscvaux.out" "testlib" "gdxrank" "gmsdesux.out" "gmsmceux.out" "scenred" "gdxtroll" "csdp" "gmsmn_ux.out" "emplib" "gams" ];
+  installPhase = with lib;
+      concatStrings [''
+        mkdir -p "$out/bin"
+        cp -a * "$out/"
+        cp ${license} "$out/gamslice.txt"
+        cp ${optgams} "$out/optgams.def"
+        ln -s $out/optgams.def $out/bin/optgams.def
+        ln -s $out/libjoatdclib64.so $out/bin/libjoatdclib64.so
+        ln -s $out/libgmdcclib64.so $out/bin/libgmdcclib64.so
+        ln -s $out/libgdxdclib64.so $out/bin/libgdxdclib64.so
+        ''
+        (concatMapStringsSep "\n" (prog: "ln -s $out/${prog} $out/bin/${prog}") binaries)
+      ];
+
+  postFixup = with lib; concatMapStringsSep "\n" (prog: "patchelf --set-interpreter \"$(cat $NIX_CC/nix-support/dynamic-linker)\" $out/${prog}") binaries;
+
+  meta = {
+    description = "General Algebraic Modeling System";
+    longDescription = ''
+      The General Algebraic Modeling System is a high-level modeling system for mathematical optimization.
+      GAMS is designed for modeling and solving linear, nonlinear, and mixed-integer optimization problems.
+    '';
+    homepage = https://www.gams.com/;
+    license = stdenv.lib.licenses.unfree;
+    maintainers = [ maintainers.scriptkiddi ];
```

Comment:
Eval fails, this should read  `[ stdenv.lib.maintainers.scriptkiddi ]` to work.

An even nicer way to fix it if you write `meta = with stdenv.lib; {` above, then you don't need to repeat that prefix for `license`, `maintainers` and `platforms`.